### PR TITLE
Upgrade rust to 1.45

### PR DESCRIPTION
**Pull request by tailhook**
Needed for https://github.com/edgedb/edgedb-cli/pull/102. Doc update https://github.com/edgedb/edgedb/pull/1554

### CODE REVIEW

**Comment by elprans** [comment]
Please rebase.

File: integration/linux/build/update.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 version="3.8"
 pipVersion="$(curl -fsSL 'https://pypi.org/pypi/pip/json' | jq -r .info.version)"
-rustVersion="1.42.0"
+rustVersion="1.44.0"
```
Comment (elprans) [inline integration/linux/build/update.sh:35]:
```suggestion
rustVersion="1.45.0"
```

**Comment by tailhook** [comment]
Yes, sorry, made some silly mess with branches locally.

**Comment by tailhook** [comment]
I'm supposed to commit generated docker files, right?

**Comment by elprans** [comment]
> I'm supposed to commit generated docker files, right?

Yes, please. We need to add a top-level `make` target to regen them easily, but `make -C integration/linux/build` should do the trick now.

**Pull request merged**